packages/embed/src/factories/create-popup/create-popup.ts

```diff
@@ -96,6 +97,7 @@ export const createPopup = (formId: string, userOptions: PopupOptions = {}): Pop
     if (isOpen(popup)) {
       popup.style.opacity = '0'
       wrapper.style.opacity = '0'
+      document.body.style.overflow = 'initial'
```

Comment:
What is the scrolling of the website was intentionally disabled (`hidden`) at the beginning? Will this preserve the behavior or will it enable scrolling regardless of its setting before opening the popup? 

Maybe we should preserve the value when opening the popup and re-use it here?